src/AspNetCoreModuleV2/CommonLib/debugutil.h

```diff
@@ -2,19 +2,75 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 #pragma once
-#define ASPNETCORE_DEBUG_FLAG_INFO          0x00000001
-#define ASPNETCORE_DEBUG_FLAG_WARNING       0x00000002
-#define ASPNETCORE_DEBUG_FLAG_ERROR         0x00000004
+#include "stdafx.h"
+#include "dbgutil.h"
 
-extern DWORD g_dwAspNetCoreDebugFlags;
+#define ASPNETCORE_DEBUG_FLAG_INFO          DEBUG_FLAG_INFO
```

Comment:
Check if there are duplicate debugutil.h files anywhere.

Reply:
Nope, I was wrong on that one.